css/index.css

```diff
@@ -101,17 +127,74 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes div {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
```

Comment:
Good job allowing the parent to determine the styles for the boxes and only overriding what has changed in each individual box.